app/views/users/sessions.py

```diff
@@ -141,6 +146,69 @@ def add_speaker_view(event_id, session_id):
         return redirect(url_for('.index_view', event_id=event_id))
 
 
+@event_sessions.route('/download_speakers_as_csv/')
+@belongs_to_event
+@can_access
+def download_speakers_as_csv(event_id):
+    sessions = DataGetter.get_sessions_by_event_id(event_id)
```

Comment:
I still don't understand why you are getting sessions instead of getting list of speakers directly for a particular event. Won't that reduce the complexity of for loops and also be more reasonable since this is a `download_speakers_as_csv` ?